src/pages/Transaction/TransactionComp.tsx

```diff
@@ -327,6 +357,80 @@ const handleCellbaseInputs = (inputs: State.Cell[], outputs: State.Cell[]) => {
   return inputs
 }
 
+export const TransactionCompLite: FC<{ transaction: State.Transaction }> = ({ transaction }) => {
+  const { hash: txHash } = useParams<{ hash: string }>()
+  const { isCellbase } = transaction
+
+  const query = useQuery(['ckb_transaction_details', txHash], async () => {
+    const ckbTransactionDetails = await fetchTransactionLiteDetailsByHash(txHash)
+    return ckbTransactionDetails.data
+  })
+
+  const transactionLiteDetails: State.TransactionLiteDetails[] = query.data ?? defaultTransactionLiteDetails
+
+  return (
+    <>
+      {transactionLiteDetails &&
+        transactionLiteDetails.map((item, index) => {
+          return (
+            <div className="transaction_lite" key={index}>
```

Comment:
Avoid using `index` as the `key`, what if using `item.address`

Ref:
- key of list: https://legacy.reactjs.org/docs/lists-and-keys.html#keys